05-how-to-contribute-to-open-source.md

```diff
@@ -144,20 +144,8 @@ You'll need to have these tools donwloaded and installed on your local machine:
 3. Clone your forked repository to your computer.
 4. Run `npm install` to install the dependencies.
 5. Create a new branch and use a descriptive name related to your contribution — for example, `feat/add-alice`.
-6. Add yourself to the guestbook.
-
-   There are two ways you can complete this activity:
-
-   - Using the CLI tool.
-   - Manually editing the `.all-contributorsrc` file.
-
-   Although adding yourself to the guestbook with the CLI tool is simpler, we recommend manually editing the file to get more experience adding and committing your changes, creating pull requests, and pull request reviews. Below, you'll find the directions for each method:
-
-   **CLI Tool**
-
-   - Run `npm run contributors:add` in your terminal.
-
-     Follow the instructions to add yourself to the guestbook. After you finish and click enter, you must click enter again to confirm your choices.
+6. Run `npm run contributors: add` in your terminal.
+   Follow the instructions to add yourself to the guestbook. After you finish and click enter, you must click enter again to confirm your choices
 
    **Manual Addition**
 
```

Comment:
Hi @Pondy007. This part needs to be omitted.

Reply:
Ok. Done : line "**Manual Addition**" removed.